apps/librepcb-cli/main.cpp

```diff
@@ -42,7 +42,7 @@ int main(int argc, char* argv[]) {
   Debug::instance();
 
   // Silence debug output, it's a command line tool
```

Comment:
Adding an explanation *why* the logging level is set to `Fatal`:

```suggestion
  // Silence logging output, it's a command line tool where logging messages
  // could lead to issues when parsing the CLI output. Real errors will be
  // printed to stderr explicitly and logging output can optionally be enabled
  // with the "--verbose" flag). But still print fatal errors since this is the
  // only way to print any error to stderr before the application gets aborted.
```